packages/create-discord-bot/src/create-discord-bot.ts

```diff
@@ -28,6 +28,12 @@ if (!directory) {
 const root = path.resolve(directory);
 const directoryName = path.basename(root);
 
+if (existsSync(root) && readdirSync(root).length > 0) {
+	console.error(chalk.red(`The directory ${chalk.yellow(`"${directoryName}"`)} is not empty.`));
+	console.error(chalk.red(`Please specify a non-empty directory.`));
```

Comment:
I don't think that was correct 😅 

```suggestion
	console.error(chalk.red(`Please specify an empty directory.`));
```